modeling/inference_models/hf.py

```diff
@@ -379,7 +380,17 @@ def init_model_config(self) -> None:
                 revision=utils.koboldai_vars.revision,
                 cache_dir="cache",
             )
+
             self.model_type = self.model_config.model_type
+
+            if "gptq_bits" in dir(self.model_config):
+                self.gptq_model = True
+                self.gptq_bits = self.model_config.gptq_bits
+                self.gptq_groupsize = self.model_config.gptq_groupsize if getattr(self.model_config, "gptq_groupsize", False) else -1
+                self.gptq_version = self.model_config.gptq_version if getattr(self.model_config, "gptq_version", False) else 1
+                self.gptq_file = None
+            else:
+                self.gptq_model = False
```

Comment:
If a GPTQ model doesn't have the gptq_bits parameter in it's config.json, as most don't, will it trigger this self.gptq_model = false statement?

Reply:
Only if it isn't also in legacy format (like having the `4bit-128g.safetensors` file)